packages/gestalt/src/Badge.js

```diff
@@ -28,5 +28,5 @@ export default function Badge({ position = 'middle', text }: Props): Node {
     [colors.darkGray]: colorSchemeName === 'darkMode',
   });
 
-  return <span className={cs}>{text}</span>;
+  return <div className={cs}>{text}</div>;
```

Comment:
This change doesn't seem to be necessary given the `inline-block` style set on the span, but in general I find reasoning about divs more intuitive than spans. Can revert this part if there's a good reason to keep this as a span.